CHANGELOG.md

```diff
@@ -3,12 +3,10 @@
 NOTE: From this version on the frameworks changed to Dynamic instead of Static library
 
 ### Bug fixes
-- 
+None.
 
 ### New Features
--  
-
-
+- Mobile SDK is now built as Dynamic Framework. [US367604]
```

Comment:
The SDK supports dynamic framework. All you need to do is update your Xcode settings.